e2e/components/Toggle/Toggle-test.avt.e2e.js

```diff
@@ -0,0 +1,61 @@
+/**
+ * Copyright IBM Corp. 2016, 2023
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+'use strict';
+
+const { expect, test } = require('@playwright/test');
+const { visitStory } = require('../../test-utils/storybook');
+
+test.describe('Toggle @avt', () => {
+  test('accessibility-checker @avt', async ({ page }) => {
```

Comment:
I guess we can remove that `@avt` text here

```suggestion
  test('accessibility-checker', async ({ page }) => {
```